cpp/oneapi/dal/chunked_array.hpp

```diff
@@ -50,6 +50,23 @@ class ONEDAL_EXPORT chunked_array : public detail::chunked_array_base {
     /// @param array Source array
     chunked_array(const chunked_array& array) : chunked_array_base{ array } {}
 
+    /// @brief       Move assignment
+    /// @param array Source array
+    chunked_array& operator=(chunked_array&& array) {
+        base_t::operator=(std::forward<base_t>(array));
+        return *this;
+    }
+
+    /// @brief       Copy assignment
+    /// @param array Source array
+    chunked_array& operator=(const chunked_array& array) {
+        base_t::operator=(array);
+        return *this;
+    }
```

Comment:
Should be equivalent but with less lines of code.

```suggestion
    chunked_array& operator=(chunked_array&& array) = default;

    /// @brief       Copy assignment
    /// @param array Source array
    chunked_array& operator=(const chunked_array& array) = default;
```